content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -129,17 +129,23 @@ Plugins wishing to perform "pre-reserve" work should use the
 NormalizeScore extension point.
 {{< /note >}}
 
-### Reserve
-
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the Node, and it exists
-to prevent race conditions while the scheduler waits for the bind to succeed.
-
-This is the last step in a scheduling cycle. Once a Pod is in the reserved
-state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
+### Reserve {#reserve}
+
+The reserve extension point happens before the scheduler actually binds a Pod to
+a node and its uses are three-fold. First, it serves as a hook to allocate
+resources in the designated node that the scheduled Pod might need.  Second, it
+serves as an informational plugin which maintain runtime state (aka "stateful
+plugins") to be notified by the scheduler when resources on a node are being
+reserved for a given Pod. Third, it exists to prevent race conditions while the
+scheduler waits for the bind to succeed. This is the penultimate step in the
+scheduling cycle.
+
+The reserve extension point may fail or succeed. If it fails, the
+[Unreserve](#unreserve) extension point is triggered. Alternatively, if it
+succeeds, the Pod is deemed to be in the reserved state, at which point,
```

Comment:
We are actually packing information pertaining to the other extension points here.

It should be more like:
"If it succeeds, the binding routine is initiated, where Permit is the first extension point to run."